tests/composite_model_test.py

```diff
@@ -539,3 +540,67 @@ def test_failure_criteria_evaluation_default_unit_system(dpf_server):
     cfc = CombinedFailureCriterion("max stress", failure_criteria=[MaxStressCriterion()])
 
     failure_container = composite_model.evaluate_failure_criteria(cfc)
+
+
+def test_composite_model_with_imported_solid_model_assembly(dpf_server):
+    """
+    Imported solid models are slightly different if compared with shell parts
```

Comment:
Provide a brief description first, followed by the more comprehensive information. The format for that brief description is to start with a brief verb ending in an s or es.